src/hotspot/share/runtime/os.cpp

```diff
@@ -1668,6 +1668,9 @@ char* os::attempt_reserve_memory_at(char* addr, size_t bytes) {
   char* result = pd_attempt_reserve_memory_at(addr, bytes);
   if (result != NULL) {
     MemTracker::record_virtual_memory_reserve((address)result, bytes, CALLER_PC);
+  } else {
+    log_info(metaspace)("Attempt to reserve memory at " INTPTR_FORMAT " for "
```

Comment:
Why is this logged for "metaspace"? I would use "os"

Reply:
Since we in this bug only concern about metaspace, if we specify -Xlog:os, it will output messaged we are not interested in from other places. I will wait for other inputs for this.

Reply:
This is 'os' code so the logging should be in the os tag space, though it could be os+metaspace. I also think info is inappropriate and this should be debug or trace level. Alternatively put the logging in a caller that is more obviously only for metaspace.

Reply:
@dholmes-ora Thanks. I changed the log to os+metaspace at debug level.